ReactWindows/ReactNative.Shared/UIManager/BaseViewManager.cs

```diff
@@ -16,7 +24,8 @@ namespace ReactNative.UIManager
     /// </summary>
     /// <typeparam name="TFrameworkElement">Type of framework element.</typeparam>
     /// <typeparam name="TLayoutShadowNode">Type of shadow node.</typeparam>
-    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> : ViewManager<TFrameworkElement, TLayoutShadowNode>
+    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
```

Comment:
After going through the details with @kevinvangelder , I agree. The "cheap" way of doing Matrix3D and Accessibility with. Live regions are totally different between UWP and WPF, with very little opportunity to share common code. It would be nice to try and share the interface, but these are all static methods. My advice is to make a Net46 copy that is a stub for now.